subprojects/core/src/main/java/org/gradle/internal/progress/ProgressFormatter.java

```diff
@@ -15,7 +15,10 @@
  */
 package org.gradle.internal.progress;
 
+import org.gradle.api.Nullable;
+
 interface ProgressFormatter {
     String incrementAndGetProgress();
     String getProgress();
+    @Nullable String getProgressIndicator();
```

Comment:
:x: I think it would be highly preferable not to split a "progress indicator" out here. Instead, we should have `BuildStatusRenderer` manage `ProgressBar`s. 

I had spiked this with other console work [here](https://github.com/gradle/gradle/commit/6115cf15d82b897411a9310ee23a0d04030ab3e6#diff-d4bc723ea2fde4a0e75ac82b6ffc4321), decided to split out the work, and never got back to it.